### DISCUSSION TITLE
Some one reprint your mod

### DISCUSSION THREAD

**Discussion by aly12767** [vectorwing/FarmersDelight#626] [General]
I found some one reprint your mod in "china edition Minecraft" and to make profits.So I want to ask if you agree this

**Comment by RaymondBlaze** [comment]
The MIT license which Farmer's Delight used allows re-distributions and modified re-distributions, with no restriction about whether it's for commercial purposes, as long as they are not claiming Farmer's Delight is their work.

I'm not expressing support for them doing so, nor do I have good feelings about Netease (the company running Minecraft Chinese Edition). All I want to say is that vectorwing actually doesn't have a valid reason to ban them from doing so, no mentioning how hard it is to exercise legal rights when the whole thing is international.

Anyways, it's still good for you to be here to notice vectorwing about the reprinting thing ;)

**Reply by aly12767** [reply]
I saw it on a short video platform this.I hope they realize that it is immoral to reprint without the consent of the author

**Comment by HaiXi2912** [comment]
我看到那是基岩版的，根据MIT协议，他们应该标注材质，玩法，全部来自农夫乐事，如果他们标注了，即使不用授权也可以使用，如果是java版直接使用农夫乐事卖钱，就算标注了农夫乐事，那也是对原作者的不尊重

**Reply by aly12767** [reply]
确实是PE的,但是没有标注,甚至觉得是他们自己的东西

**Reply by aly12767** [reply]
现在才发现网易java版好像也有转载的
